tools/myst/enc/enc.c

```diff
@@ -628,6 +633,40 @@ int myst_enter_ecall(
     return ret;
 }
 
+#define ENTER_STACK_SIZE (132 * 1024)
```

Comment:
Need a comment here, like "This is the stack size of the main thread."

Reply:
Good idea. Done.